shell/browser/api/electron_api_web_frame_main.cc

```diff
@@ -316,13 +348,39 @@ void WebFrameMain::Connect() {
   }
 }
 
+content::FindRequestManager* WebFrameMain::GetOrCreateFindRequestManager() {
+  if (!find_request_manager_.get()) {
+    // No existing FindRequestManager found, so one must be created.
+    auto* web_contents = static_cast<content::WebContentsImpl*>(
+        content::WebContents::FromRenderFrameHost(render_frame_));
+    find_request_manager_ = std::make_unique<content::FindRequestManager>(
+        web_contents,
+        static_cast<content::RenderFrameHostImpl*>(render_frame_));
+  }
+  // Concurrent find sessions must not overlap, so destroy any existing
+  // FindRequestManagers in any inner WebFrameMain.
+  for (content::RenderFrameHost* rfh : render_frame_->GetFramesInSubtree()) {
```

Comment:
If a find session exists in a parent frame, would that be an issue? If so, this might need to be `render_frame_->GetMainFrame()->GetFramesInSubtree()`.

Reply:
Good catch, yup it should be `render_frame_->GetMainFrame()->GetFramesInSubtree()`